src/ForkCMS/Bundle/InstallerBundle/Resources/views/Installer/step1.html.twig

```diff
@@ -36,59 +36,59 @@
               <h3>PHP Extensions</h3>
               <div class="list-group">
                 <div class="list-group-item list-group-item-{{ checker.errors.extensionCURL }}">
-                  <h4 class="list-group-item-heading">cURL</h4>
+                  <h3 class="list-group-item-heading">cURL</h3>
```

Comment:
I like them smaller, but the h3 is styled smaller then the h4 which is not correct.
I will fix that also.